test/parallel/test-http-set-timeout-server.js

```diff
@@ -169,6 +165,47 @@ test(function idleTimeout(cb) {
       allowHalfOpen: true,
     };
     const c = net.connect(options, () => {
+      c.on('error', (e) => {
+        if (e.message !== 'read ECONNRESET')
+          throw e;
+      });
+      c.write('GET /1 HTTP/1.1\r\nHost: localhost\r\n\r\n');
+      // Keep-Alive
+    });
+  }));
+});
+
+test(function fastTimeout(cb) {
+  let connectionHandlerInvoked = false;
+  let timeoutHandlerInvoked = false;
+  let connectionSocket;
+
+  function invokeCallbackIfDone() {
+    if (connectionHandlerInvoked && timeoutHandlerInvoked) {
+      connectionSocket.destroy();
+      server.close();
+      cb();
+    }
+  }
+
+  const server = http.createServer(common.mustCall((req, res) => {
+    req.on('timeout', common.mustNotCall());
+    res.end();
```

Comment:
It might be emitted so `common.mustNotCall()` was wrong. See https://github.com/nodejs/node/pull/14380#issuecomment-317152594.